NuRadioMC/SignalProp/propagation.py

```diff
@@ -18,6 +18,9 @@ def get_propagation_module(name='analytic'):
     elif(name=='direct_ray'):
         from NuRadioMC.SignalProp.directRayTracing import directRayTracing
         return directRayTracing
+    elif(name=='radiopropa'):
+        from NuRadioMC.SignalProp.Simple_radiopropa_tracer import ray_tracing
```

Comment:
You should give the radioPropa raytracing class a more unique name, this can very easily lead to problems if you make an error with the imports.

Reply:
so not import ray_tracing but ray_tracing_radiopropa? Shouldn't this then also be the case for the analytic one?

Reply:
Yes, you are right. But I guess back then analytic raytracing was the only one. Simpler times ;-)